aip/general/0121.md

```diff
@@ -94,6 +94,37 @@ the standard methods. Custom methods offer the same design freedom as
 traditional RPC APIs, which can be used to implement common programming
 patterns, such as database transactions, import and export, or data analysis.
 
+### Strong Consistency
+
+For methods that operate on the [management plane][], the completion of those
+operations (either successful or with an error, LRO or synchronous) **must**
+mean that the state of the resource's existence and all user-settable values
+have reached a steady-state.
+
+[output only][] values **should** also have reached a steady-state, except
+for values that are related to the resource [state][].
+
+Examples include:
+
+- Following a successful create, a get request for a resource **must**
```

Comment:
Unless there's a race condition whereby it got deleted immediately. Same below for update.

If two people are updating, (unless the service handles stale state), then last one will win.

Reply:
good call! I updated the working to state that it has to be the latest mutation on the resource (should clarify that last operation wins). WDYT?